### PR TITLE
s/undervote/skippedrank as appropriate in code, config, comments, and UI

### LINKED ISSUE #718: Clarify Undervotes vs Skipped Votes in UI, config files, and comments
See comments in #633 

The phrase "undervotes" is used to mean both "voter did not rank anything" and "voter skipped too many rankings."

Clean this up.

### PR DESCRIPTION
**Pull request by artoonie**
Closes #718

### CODE REVIEW

File: config/spotbugs/exclude.xml [hunk 1]
```diff
@@ -16,6 +16,10 @@
     <Method name="readCastVoteRecord"/>
     <Bug code="NP"/>
   </Match>
+  <Match>
+    <Class name="network.brightspots.rcv.RawContestConfig$CvrSource"/>
+    <Bug code="UwF"/>
+  </Match>
```
Comment (tarheel) [inline config/spotbugs/exclude.xml:22]: What is this?

File: src/main/java/network/brightspots/rcv/GuiConfigController.java [hunk 2]
```diff
@@ -1156,7 +1156,14 @@ public LocalDate fromString(String string) {
...
     );
-
+    tableColumnCvrContestId.setCellValueFactory(new PropertyValueFactory<>("contestId"));
+    tableColumnCvrOvervoteLabel.setCellValueFactory(new PropertyValueFactory<>("overvoteLabel"));
+    tableColumnCvrSkippedRankLabel
+        .setCellValueFactory(new PropertyValueFactory<>("skippedRankLabel"));
+    tableColumnCvrUndeclaredWriteInLabel
+        .setCellValueFactory(new PropertyValueFactory<>("undeclaredWriteInLabel"));
+    tableColumnCvrTreatBlankAsUndeclaredWriteIn
+        .setCellValueFactory(new PropertyValueFactory<>("treatBlankAsUndeclaredWriteIn"));
```
Comment (tarheel) [inline src/main/java/network/brightspots/rcv/GuiConfigController.java:1166]: could you clarify why these changes are in this PR and what they do?

File: src/main/java/network/brightspots/rcv/RawContestConfig.java [hunk 3]
```diff
@@ -80,10 +80,13 @@ public static class CvrSource {
     private SimpleStringProperty overvoteDelimiter = new SimpleStringProperty();
     private SimpleStringProperty provider = new SimpleStringProperty();
     private SimpleStringProperty overvoteLabel = new SimpleStringProperty();
-    private SimpleStringProperty undervoteLabel = new SimpleStringProperty();
+    private SimpleStringProperty skippedRankLabel = new SimpleStringProperty();
     private SimpleStringProperty undeclaredWriteInLabel = new SimpleStringProperty();
     private SimpleBooleanProperty treatBlankAsUndeclaredWriteIn = new SimpleBooleanProperty();
 
+    // Deprecated fields
```
Comment (tarheel) [inline src/main/java/network/brightspots/rcv/RawContestConfig.java:87]: Oh, wow. Is this our first one ever? Or I guess we just didn't bother with logic like this before? E.g. I think `overvoteLabel` used to be config-level and then we changed it to source-level.

File: src/main/java/network/brightspots/rcv/Tabulator.java [hunk 4]
```diff
@@ -1072,7 +1072,7 @@ && isCandidateContinuing(cvr.getCurrentRecipientOfVote())) {
           continue;
         }
 
-        // the current ranking is not an overvote or undervote
+        // the current ranking is not inactive by overvote or skipped ranking
```
Comment (tarheel) [inline src/main/java/network/brightspots/rcv/Tabulator.java:1075]: maybe clarify to "... or too many skipped rankings"

File: src/test/resources/network/brightspots/rcv/test_data/2013_minneapolis_mayor/2013_minneapolis_mayor_config.json [hunk 5]
```diff
@@ -19,7 +19,7 @@
     "provider" : "ess",
     "treatBlankAsUndeclaredWriteIn": false,
     "overvoteLabel": "overvote",
-    "undervoteLabel": "undervote",
+    "skippedRankLabel": "undervote",
```
Comment (tarheel) [inline src/test/resources/network/brightspots/rcv/test_data/2013_minneapolis_mayor/2013_minneapolis_mayor_config.json:22]: not a big deal, but ideally we would change all of these test config "undervote" values to "skipped rank" or whatever

File: config/spotbugs/exclude.xml (hunk 1 above)
Comment (artoonie) [reply config/spotbugs/exclude.xml:22]: spotbugs thinks `getUndervoteLabel` is unused, but it's used via reflection when jackson deserializes

File: src/main/java/network/brightspots/rcv/RawContestConfig.java (hunk 3 above)
Comment (artoonie) [reply src/main/java/network/brightspots/rcv/RawContestConfig.java:87]: First one in this class, but at the bottom of this file, you'll notice `ContestRules` has the same pattern

File: src/main/java/network/brightspots/rcv/GuiConfigController.java (hunk 2 above)
Comment (artoonie) [reply src/main/java/network/brightspots/rcv/GuiConfigController.java:1166]: I think github is confused!! I don't think these are actually in this PR -- let me merge develop and see what happens. Not sure why this is happening. These are part of editable table changes.

Comment (artoonie) [reply src/main/java/network/brightspots/rcv/GuiConfigController.java:1166]: Merging develop didn't help, but locally, `git blame` shows that you added these lines in 2020 and the only changes are to `tableColumnCvrSkippedRankLabel`. I'm very confused here too -- I merged develop and it didn't go away. Is this a github bug or is there something I'm missing? If it's a github bug, it'll make reviewing very hard...

File: src/test/resources/network/brightspots/rcv/test_data/2013_minneapolis_mayor/2013_minneapolis_mayor_config.json (hunk 5 above)
Comment (artoonie) [reply src/test/resources/network/brightspots/rcv/test_data/2013_minneapolis_mayor/2013_minneapolis_mayor_config.json:22]: Agree, downside is that we'd have to update all the CVRs, and those are XLSXs, which makes it hard to just `sed` find/replace each instance of `undervote`

File: src/main/java/network/brightspots/rcv/GuiConfigController.java (hunk 2 above)
Comment (tarheel) [reply src/main/java/network/brightspots/rcv/GuiConfigController.java:1166]: Bizarre. No idea!

**Review by tarheel — APPROVED** [review]
